### ISSUE TITLE
[7.5.1] yarn build_development: Module not found: Error: Can't resolve 'datatables.net'

### ISSUE THREAD

**Issue by Gianluke** [archesproject/arches#10731]
Hi,
I followed the official installation manual of Arches core 7.5.1 (on dedicate Ubuntu 20.04 LTS VM with "scripted dependency" installation) but when I launch the "yarn build_development" command I encounter the following errors:

> ERROR in ./media/node_modules/datatables.net-bs/js/dataTables.bootstrap.min.js 
> Module not found: Error: Can't resolve 'datatables.net' in '/home/arches/myproject/myproject/media/node_modules/datatables.net-bs/js'
>  @ ./media/node_modules/datatables.net-buttons-bs/js/buttons.bootstrap.min.js
>  @ ../../ENV/lib/python3.10/site-packages/arches/app/media/js/bindings/datatable.js
> 
> ERROR in ./media/node_modules/datatables.net-buttons/js/buttons.html5.min.js 
> Module not found: Error: Can't resolve 'datatables.net' in '/home/arches/myproject/myproject/media/node_modules/datatables.net-buttons/js'
>  @ ../../ENV/lib/python3.10/site-packages/arches/app/media/js/bindings/datatable.js
> 
> ERROR in ./media/node_modules/datatables.net-buttons/js/buttons.print.min.js 
> Module not found: Error: Can't resolve 'datatables.net' in '/home/arches/myproject/myproject/media/node_modules/datatables.net-buttons/js'
>  @ ../../ENV/lib/python3.10/site-packages/arches/app/media/js/bindings/datatable.js
> 
> ERROR in ./media/node_modules/datatables.net-buttons/js/dataTables.buttons.min.js 
> Module not found: Error: Can't resolve 'datatables.net' in '/home/arches/myproject/myproject/media/node_modules/datatables.net-buttons/js'
>  @ ./media/node_modules/datatables.net-buttons/js/buttons.html5.min.js
>  @ ../../ENV/lib/python3.10/site-packages/arches/app/media/js/bindings/datatable.js
> 
> ERROR in ./media/node_modules/datatables.net-responsive/js/dataTables.responsive.js 
> Module not found: Error: Can't resolve 'datatables.net' in '/home/arches/myproject/myproject/media/node_modules/datatables.net-responsive/js'
>  @ ./media/node_modules/datatables.net-responsive-bs/js/responsive.bootstrap.js
>  @ ../../ENV/lib/python3.10/site-packages/arches/app/media/js/bindings/datatable.js

I also tried to carry out the installation process with python versions 3.9, 3.10 and 3.11 but still the same problem.

Any suggestions?

Thank's in advance.

**Comment by jacobtylerwalls** [comment]
@chiatt plans to apply fixes for this to 7.5.x and earlier.

In the meantime, in your project's package.json, you can force yarn to use an older version of `datatables.net` by adding this:
```json
    "resolutions": {
        "datatables.net": "1.12.1"
    },
```
Then remove yarn.lock and repeat the yarn install and yarn build_... commands.

**Comment by Gianluke** [comment]
Thank you, resolved ;).
